manifests/s/Stellar/StellarCLI/22.0.0-rc.1/Stellar.StellarCLI.locale.en-US.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Stellar.StellarCLI
+PackageVersion: 22.0.0-rc.1
+PackageLocale: en-US
+Publisher: Stellar
+PublisherUrl: https://stellar.org/
+PackageName: Stellar CLI
+PackageUrl: https://github.com/stellar/stellar-cli
+License: Apache 2.0
+LicenseUrl: https://github.com/stellar/stellar-cli/blob/main/LICENSE
+ShortDescription: CLI for Stellar developers
+Description: CLI for Stellar developers
+Moniker: soroban
```

Comment:
Unclear why we need to mention soroban here. Soroban is the execution runtime that the Stellar network uses. It's not built into the Stellar-CLI, and the Stellar-CLI is primarily about interacting with the Stellar network, not the execution runtime. A while ago the CLI carried the name "soroban-cli" due to historical hang over.